tempesta_fw/client.c

```diff
@@ -75,6 +78,7 @@ tfw_client_put(TfwClient *cli)
 
 	TFW_DBG("free client: cli=%p\n", cli);
 	kmem_cache_free(cli_cache, cli);
+	atomic64_dec(&act_cli_n);
```

Comment:
By the way, is it possible to reuse `clnt.online` for the same purpose?

Reply:
Yes, technically possible, but I thought that it is not worth to mix closing synchronization with pure statistic variables.